### PR TITLE
Added Python to Travis build and clarified setup instructions.

### PR DESCRIPTION
**Pull request by haberman**
@xfxyjwf 

Hopefully this PR will pass right away in Travis, but if not I'll iterate and fix it. :)

### CODE REVIEW

**Comment by haberman** [comment]
Ok looks like tests are passing. :)

File: python/README.md [hunk 1]
```diff
@@ -100,12 +111,7 @@ To use the C++ implementation, you need to:
 2) Export an environment variable:
 
     $ export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=cpp
-    $ export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION_VERSION=2
 
-You need to export this variable before running setup.py script to build and
-install the extension.  You must also set the variable at runtime, otherwise
+You must set this variable at runtime, before running your program, otherwise
```
Comment (xfxyjwf) [inline python/README.md]: So the flag is not needed when running the tests?

Comment (haberman) [inline python/README.md]: Exactly, the tests set it automatically when --cpp_implementation is specified:
https://github.com/google/protobuf/blob/master/python/setup.py#L162

**Comment by xfxyjwf** [comment]
LGTM

**Pull request merged**